### ISSUE TITLE
[REQUEST] Limit on how much bots and proxy can run at once

### ISSUE THREAD

**Issue by crazygamer96** [openbullet/OpenBullet2#489]
**Is your feature request related to a problem? Please describe.**
Proxies getting banned 

**Describe the solution you'd like**
Not sure if this is already a thing that I just can't find in the settings but it would be nice to have an option that limits how much bots a proxy can be running at once because sometimes not sure why the job likes to prefer one proxy and tries to that like 40 bots with 1 proxy and just instantly gets my proxy banned from the website I am using

**Comment by JReming85** [comment]
Make sure Shuffle proxies is ticked
Never ban proxies is unticked
Concurrent proxy mode is unticked (most important)

![image](https://user-images.githubusercontent.com/21246660/134753525-89807df1-fc6d-4b69-8cf2-96b48062d248.png)

**Comment by crazygamer96** [comment]
Yeah I have all those settings and figured out why my proxies were getting banned (I forgot to untick ban if no match on a keycheck) but I still think this would be a great feature (as a safety measure) because the website I use has a limit and only allows me to do like 5 bots per proxy and I only have like 79 proxies so I can't really go above or else its going to start using one proxy on multiple bots which leads to them getting banned

**Comment by JReming85** [comment]
Technically it will only reuse a proxy multiple times if the **Concurrent proxy mode** is enabled.  I have yet to see it use the same proxy in the preview/status page at the same time without that enabled.

That said my pools are pretty large
